Update DuraCloud integration page

sallain opened the pull request:
A general clean-up for the DuraCloud integration page.

After discussion with @ablwr and @dcraze, I have removed the sections
related to using DuraCloud as a transfer source location. Though it is
possible, early tests following the development of the integration
showed that using DuraCloud for this purpose was very slow and possibly
unreliable. Artefactual does not recommend this for Archives Direct
users, so it seemed better to remove the information. It can be added
back if that functionality improves.

This is connected to archivematica/Issues#171

sallain committed:
Update DuraCloud integration page

A general clean-up for the DuraCloud integration page.

After discussion with @ablwr and @dcraze, I have removed the sections
related to using DuraCloud as a transfer source location. Though it is
possible, early tests following the development of the integration
showed that using DuraCloud for this purpose was very slow and possibly
unreliable. Artefactual does not recommend this for Archives Direct
users, so it seemed better to remove the information. It can be added
back if that functionality improves.

This is connected to archivematica/Issues#171

ablwr commented on admin-manual/installation-setup/integrations/duracloud-setup.rst:
```diff
@@ -1,226 +1,107 @@
+#. In the Storage Service's Spaces tab, click **Create new space** and enter the
+   following information:
+
+   * **Access protocol**: DuraCloud
```
Should this section just link to the Storage Service docs instead? https://www.archivematica.org/en/docs/storage-service-0.15/administrators/#duracloud

ablwr replied:
(or should this information be moved and updated there?, is the better question)

sallain replied:
Good point @ablwr, thanks! I just went with what was already on the page but pointing out to the SS docs to reduce the number of places we need to keep updated is better.

sallain committed:
Updated to point to SS docs

sallain committed:
Add explicit reference to leading slash

I've added an explicit reference telling users to make sure that they are
copying the DuraCloud space name exactly, ensuring that they're not
including a leading slash.

sallain commented:
@ross-spencer I've included an explicit reference to making sure that the space name doesn't include a leading slash.

ross-spencer commented on admin-manual/installation-setup/integrations/duracloud-setup.rst:
```diff
@@ -1,226 +1,101 @@
+   DuraCloud, you should create two spaces by repeating the instructions in Step
+   1.
+
+#. In the Storage Service, navigated to the **Spaces** tab. You should see all
```
```suggestion
#. In the Storage Service, navigate to the **Spaces** tab. You should see all
```

(or `once navigated to`)

ross-spencer commented on admin-manual/installation-setup/integrations/duracloud-setup.rst:
```diff
@@ -1,226 +1,101 @@
+<storageservice:location-purposes>` within Archivematica:
 
-Two DuraCloud users should be created:
+* **AIP storage**: A DuraCloud space where completed AIPs are placed for
+  long-term storage.
+* **DIP storage**: A DuraCloud space where DIPs can be stored. Note that it is
+  not necessary to store DIPs if you are sending them directly to an access
+  system.
+* **Transfer backlog**: A DuraCloud space used by Archivematica for material
+  sent to the :ref:`backlog <backlog>` during processing.
 
```
I wonder if we can note that this is a recommendation, i.e. there are no technical reasons all three couldn't be `<a bucket>`. In doing this I feel it helps folk with the logic of what they're doing, as well as the logic of something that might go wrong e.g. if accidentally set up to be one bucket, then that's because it's all feasible. And we could add a lint of digipres and records and info mgmt too -- something along the lines of, management of these package types are best done separated into multiple bucket types dedicated to a storage service's purpose.

sallain replied:
Thanks, I've included a note to that effect.

ross-spencer commented on admin-manual/installation-setup/integrations/duracloud-setup.rst:
```diff
@@ -1,226 +1,101 @@
+
+   * **Purpose**: The purpose of the space (AIP storage, DIP storage, or
+     transfer backlog.)
+   * **Pipeline**: Click on the pipeline name so that it is highlighted orange.
```
Accessibility: perhaps we can just say highlighted rather than orange? My own point of reference is that in my dark scheme I see largely in different colors, but other folk might have different barriers. (Also, in regular mode I didn't know this should be orange for me!) 

![image](https://user-images.githubusercontent.com/1880412/73556311-33adf400-441d-11ea-9036-4b7e15c689b6.png)

sallain replied:
Excellent point!

ross-spencer reviewed (commented):
This looks good Sara. I have left comments and suggestions but I also think technically it looks fine. 

What are your plans for the docs? This feels like a bit of a watershed where the storage service pages are more an appendix than anything? I do recognize that our choice to describe all space protocols on a single page there holds us back somewhat in terms of expression. Maybe we should bring them all together finally? (In time, in a managed way, etc. etc.)

sallain committed:
Update admin-manual/installation-setup/integrations/duracloud-setup.rst

Co-Authored-By: Ross Spencer <all.along.the.watchtower2001@gmail.com>

sallain committed:
Incorporate feedback and rebase
